docs/validation/6_multirotor_sims.tex

```diff
@@ -0,0 +1,216 @@
+\end{figure} 
+
+\begin{figure}[H]
+\centering
+\includegraphics[width=16.5cm]{img/600g_motor.png}
+\caption{Motor Characteristics for 600 g Model Weight}
+\end{figure} 
+\clearpage
+
+\subsection{Payload Weight Variations}
+As the client may wish to extend the functionality of the multirotor by adding or replacing components of the computing platform or multirotor, simulations have been performed to provide insight as to the impact of payload weight on flight performance. In particular, the following simulations measure maximum hover time across differing model weights, battery configurations (ex. 3S/4S), and battery capacities. All other (non-weight-related) parameters are kept constant across tests.
+
+\begin{figure}[H]
+\centering
+\begin{subfigure}{.5\textwidth}
+\begin{tikzpicture}
+\pgfplotsset{compat=1.3}
+\centering
+\begin{axis}[
+  xlabel=Total Model Weight (grams),
+  ylabel=Hover Time (minutes),
+  ylabel shift = -5 pt]
+\addplot table [y=flightduration, x=weight]{plots/3S-5000.txt};
+\addlegendentry{5000 mAh}
+\addplot table [y=flightduration, x=weight]{plots/3S-6000.txt};
+\addlegendentry{6000 mAh}
+\addplot table [y=flightduration, x=weight]{plots/3S-10000.txt};
+\addlegendentry{10000 mAh}
+\end{axis}
+\end{tikzpicture}
+\caption{Simulated Hover Times for 3S Batteries}
+\end{subfigure}%
+\begin{subfigure}{.5\textwidth}
+\centering
+\begin{tikzpicture}
+\pgfplotsset{compat=1.3}
+\begin{axis}[
+  xlabel=Total Model Weight (grams),
+  ylabel=Hover Time (minutes),
+  ylabel shift = -5 pt]
+\addplot table [y=flightduration, x=weight]{plots/4S-5000.txt};
+\addlegendentry{5000 mAh}
+\addplot table [y=flightduration, x=weight]{plots/4S-6000.txt};
+\addlegendentry{6000 mAh}
+\addplot table [y=flightduration, x=weight]{plots/4S-10000.txt};
+\addlegendentry{10000 mAh}
+\end{axis}
+\end{tikzpicture}
+\caption{Simulated Hover Times for 4S Batteries}
+\end{subfigure}
+\end{figure}
+\subsection{Propulsion Performance Exploration}
+\label{prop_explore}
+
+In order to increase the effective range and flight duration of the multirotor, the client may wish to install higher-capacity batteries (or alter their configurations). The following simulations sweep potential battery capacities and outline their impact on hover time, mixed flight time, and weight capacities.
+
+\begin{figure}[H]
+\centering
+\begin{subfigure}{.5\textwidth}
+\begin{tikzpicture}[scale=0.95]
+\pgfplotsset{compat=1.3}
+\centering
+\begin{axis}[
+  xlabel=Battery Capacity (mAh),
+  ylabel=Flight Duration (minutes),
+  ylabel shift = -5 pt,
+  legend pos=south east]
+\addplot table [y=hover, x=capacity]{plots/3S-time.txt};
+\addlegendentry{Hover Flight}
+\addplot table [y=mixed, x=capacity]{plots/3S-time.txt};
+\addlegendentry{Mixed Flight}
+\end{axis}
+\end{tikzpicture}
+\caption{3S Batteries}
+\end{subfigure}%
+\begin{subfigure}{.5\textwidth}
+\centering
+\begin{tikzpicture}[scale=0.95]
+\pgfplotsset{compat=1.3}
+\centering
+\begin{axis}[
+  xlabel=Battery Capacity (mAh),
```

Comment:
The output graph in the PDF has a little "•10^4" in the corner. If the y axes look too busy, do you think we should change the x scale to be in Ah instead of mAh?

Reply:
pgfplots did this for me automatically. I agree, I'll update to show Ah.